packages/console/src/pages/EnterpriseSsoDetails/Connection/index.tsx

```diff
@@ -63,7 +63,7 @@ function Connection<T extends SsoProviderName>({ isDeleted, data, onUpdated }: P
 
       const updatedSsoConnector = await api
         .patch(`api/sso-connectors/${ssoConnectorId}`, {
-          json: { config: cleanDeep(formData) },
+          json: { config: cleanDeep(formData, { emptyObjects: false, undefinedValues: false }) },
```

Comment:
should we clean the undefined fields under config?

Reply:
We should since this is a replace of old `config` value?

Reply:
Updated, for `config` we do not need to change the `cleanDeep` options.